storage/innobase/buf/buf0flu.cc

```diff
@@ -139,6 +139,30 @@ inline void buf_pool_t::page_cleaner_wakeup()
   {
     page_cleaner_is_idle= false;
     pthread_cond_signal(&do_flush_list);
+  } else if (pct_lwm != 0.0 && dirty_pct != 0) {
+    /* dirty_pct < srv_max_dirty_pages_pct_lwm so it could be idle flushing
```

Comment:
We are trying to move to the common formatting style, where lines are split before `{` or after `}`. An exception is the opening brace after `switch (...) {`.

I would prefer to refer to user-visible parameters in comments or commit messages. It is a bit unfortunate that we have this `srv_` obfuscation layer in the internal variable names. So, I would prefer to write `innodb_max_dirty_pages_pct_lwm` in the comment.